Lib/collections/__init__.py

```diff
@@ -961,6 +961,24 @@ def clear(self):
         'Clear maps[0], leaving maps[1:] intact.'
         self.maps[0].clear()
 
+    def __ior__(self, other):
+        self.maps[0].update(other)
```

Comment:
I think this ought to be `self.maps[0] |= other` -- most other ChainMap methods also defer to the corresponding method on `self.maps[0]`.